test/extensions/filters/network/redis_proxy/config_test.cc

```diff
@@ -89,6 +98,15 @@ stat_prefix: foo
 
 TEST(RedisProxyFilterConfigFactoryTest,
      DEPRECATED_FEATURE_TEST(RedisProxyCorrectProtoLegacyCatchAllCluster)) {
+  TestScopedRuntime scoped_runtime;
+  Runtime::LoaderSingleton::getExisting()->mergeValues(
+      {{"envoy.deprecated_features:envoy.config.filter.network.redis_proxy.v2.RedisProxy."
+        "PrefixRoutes.catch_all_cluster",
+        "true"}});
+  Runtime::LoaderSingleton::getExisting()->mergeValues(
+      {{"envoy.deprecated_features:envoy.extensions.filters.network.redis_proxy.v3alpha.RedisProxy."
+        "PrefixRoutes.hidden_envoy_deprecated_cluster",
+        "true"}});
```

Comment:
This weirdness is because we're loading deprecated v2 config with a v3 proto.  Arguably we should have left the deprecated feature test using a v2 proto rather than envoy::extensions::filters::network::redis_proxy::v3alpha::RedisProxy
Can fix if we care but I'm pretty convinced we don't require both for v3 by the intergration tests not neeting it.

Reply:
Probably fine for this unit test, we tend to make v3alpha everywhere for unit tests.